client/src/lib/dependency-injection/graphql/helpers.js

```diff
@@ -26,20 +35,14 @@ export const getVariables = ({ params, pagination = true }) => {
 
 export const getParams = ({ params, pagination = true }) => {
   const items = (pagination) ? { ...params, ...paginationFields } : params;
-  const paramList = Object.entries(items)
-    .map(([paramName, varName]) => (
-      `${paramName}: $${varName}`
+  const paramList = Object.keys(items)
+    .map((paramName) => (
+      `${paramName}: $${paramName}`
     ));
 
   return paramList.length ? `(${paramList.join(', ')})` : '';
 };
```

Comment:
This was resulting in nonsensical queries like
```graphql
query ReadNotes($param: SomeInputType!) {
  readNotes(param: $SomeInputType!) {
    content
    id
  }
}
```
which is obviously incorrect - it should be
```graphql
query ReadNotes($param: SomeInputType!) {
  readNotes(param: $param) {
    content
    id
  }
}
```
which is what it now does.